### PR TITLE
fix: swap out css module prompt with lottery factor

### PR DESCRIPTION
**Pull request by bdougie**
## Description

The css modules example was flawed because that is not something matters or changes often in turbo. Instead, I'd rather include a question that highlights th work @nickytonline just shipped.

## Steps to QA

1. Click the lottery factor prompt.

## Tier (staff will fill in)

- [ ] Tier 1
- [ ] Tier 2
- [ ] Tier 3
- [x] Tier 4

## [optional] What gif best describes this PR or how it makes you feel?

giphy (25)

### CODE REVIEW

**Comment by bdougie** [comment]
using copilot workspaces to fix the box sizes. 

Screenshot 2024-05-16 at 5 12 30 PM

**Review by nickytonline — APPROVED** [review]
:shipit:

File: pages/star-search/index.tsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ const SUGGESTIONS = [
   },
   {
     title: "Find contributors based on their work",
-    prompt: "Who are people making pull requests in vercel/turbo about css modules?",
+    prompt: "What is the lottery factor of remix/react-router?",
```
Comment (bdougie) [inline pages/star-search/index.tsx:49]:
```suggestion
    prompt: "Show me the lottery factor for contributors in the remix/react-router project?",
```

File: pages/star-search/index.tsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ const SUGGESTIONS = [
   },
   {
     title: "Find contributors based on their work",
-    prompt: "Who are people making pull requests in vercel/turbo about css modules?",
+    prompt: "Show me the lottery factor for contributors in the remix/react-router project?",
```
Comment (bdougie) [inline pages/star-search/index.tsx:49]:
```suggestion
    prompt: "Show me the lottery factor for contributors in the remix-run/react-router project?",
```

**Comment by bdougie** [comment]
@BekahHW with this change we can focus on these 4 prompts for launch.

**Review by BekahHW — APPROVED** [review]

**Pull request merged**